scipy/stats/tests/test_multivariate.py

```diff
@@ -1964,6 +1965,40 @@ def test_to_corr(self):
         assert_allclose(m[0,0], 1)
 
 
+class TestRandomDirection:
+    @pytest.mark.parametrize("dim", [1, 2, 3, 4])
```

Comment:
```suggestion
    @pytest.mark.parametrize("dim", [1, 3])
```
Most cases are no longer needed since the special cases were removed.